docs/API.md

```diff
@@ -60,14 +63,36 @@ type Step = (page: Page, options?: Options) => void;
 ```
 
 <a name="bootstrap"><a/>
-## `dappeteer.bootstrap(puppeteerLib: typeof puppeteer, options: OfficialOptions & MetaMaskOptions): Promise<[Dappeteer, Page, Browser]>`
+## `dappeteer.bootstrap(options: DappeteerLaunchOptions & MetaMaskOptions): Promise<{
+  dappeteer: Dappeteer;
+  browser: DappeteerBrowser;
+  page: DappeteerPage;
+}>`
 ```typescript
-interface OfficialOptions {
-  metaMaskVersion: 'latest' | string;
+type DappeteerLaunchOptions = {
+  metaMaskVersion?:
+    | typeof RECOMMENDED_METAMASK_VERSION
```

Comment:
We don't need to specify this in docks

Reply:
You mean the `RECOMMENDED_METAMASK_VERSION` in particular right? I removed this, the rest is quite useful for JS devs I believe.

Reply:
yup